orion/primitives/jsons/orion.primitives.tadgan.TadGAN.json

```diff
@@ -1,356 +1,373 @@
-                      "parameters": {
-                          "rate": 0.25
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Conv1D",
-                      "parameters": {
-                          "filters": 64,
-                          "kernel_size": 5
-                      }
-                  },
-                  {
-                      "class": "keras.layers.advanced_activations.LeakyReLU",
-                      "parameters": {
-                          "alpha": 0.2
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Dropout",
-                      "parameters": {
-                          "rate": 0.25
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Flatten",
-                      "parameters": {}
-                  },
-                  {
-                      "class": "keras.layers.Dense",
-                      "parameters": {
-                          "units": 1
-                      }
-                  }
-              ]
-          },
-          "layers_critic_z": {
-              "type": "list",
-              "default": [
-                  {
-                      "class": "keras.layers.Flatten",
-                      "parameters": {}
-                  },
-                  {
-                      "class": "keras.layers.Dense",
-                      "parameters": {
-                          "units": "dense_units"
-                      }
-                  },
-                  {
-                      "class": "keras.layers.advanced_activations.LeakyReLU",
-                      "parameters": {
-                          "alpha": 0.2
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Dropout",
-                      "parameters": {
-                          "rate": 0.2
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Dense",
-                      "parameters": {
-                          "units": "dense_units"
-                      }
-                  },
-                  {
-                      "class": "keras.layers.advanced_activations.LeakyReLU",
-                      "parameters": {
-                          "alpha": 0.2
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Dropout",
-                      "parameters": {
-                          "rate": 0.2
-                      }
-                  },
-                  {
-                      "class": "keras.layers.Dense",
-                      "parameters": {
-                          "units": 1
-                      }
-                  }
-              ]
-          }
-      }
-  }
+    "name": "orion.primitives.tadgan.TadGAN",
```

Comment:
Can you double check the style — 2 spaces or 4 spaces — for json file?

Reply:
`lstm_autoencoder.json` and `lstm_dynamic_threshold.json` both seem to use 4 spaces for tabs